test/integration-test/src/tests/load.rs

```diff
@@ -145,6 +149,26 @@ fn unload_xdp() {
     assert_unloaded("pass");
 }
 
+#[test]
+fn test_loaded_at() {
+    let mut bpf = Bpf::load(crate::TEST).unwrap();
+    let prog: &mut Xdp = bpf.program_mut("pass").unwrap().try_into().unwrap();
+    let load_time = SystemTime::now();
+    prog.load().unwrap();
+    assert_loaded("pass");
+
+    let loaded_at = prog.info().unwrap().loaded_at();
+
+    // make sure loaded_at() api is correct with a millisecond tolerance.
```

Comment:
Wouldn't it be more robust to:
- get current time t1
- load
- get current time t2
- assert load is been t1 and t2
?